analytics-forms/src/main/java/com/liferay/analytics/client/android/forms/FocusChangeObservable.kt

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.analytics.client.android.forms
+
+import android.util.Pair
+
+import android.view.View
+
+import io.reactivex.Observable
+import io.reactivex.Observer
+import io.reactivex.android.MainThreadDisposable
+
+import java.sql.Timestamp
+
+/**
+ * @author Igor Matos
+ */
+class FocusChangeObservable(private val _view: View) : Observable<Pair<Boolean, Long>>() {
+
+	override fun subscribeActual(observer: Observer<in Pair<Boolean, Long>>) {
+
+		val listener = FocusChangeObservable.Listener(_view, observer)
+
+		observer.onSubscribe(listener)
```

Comment:
I don't love having to subscribe from library code because dealing with memory and unsubscriptions is hard but I can see the case.

Any way of unsubscribe?